src/pages/community.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Community
+description:
+  This page contains a curated list of interesting Front-Commerce community
+  projects and resources.
+---
+
+<p>{frontMatter.description}</p>
+
+import ContactLink from "@site/src/components/ContactLink";
+
+Please <ContactLink>let us know</ContactLink> if you want to share a project
+here.
+
+# Examples
+
+For common tasks, or more specific examples, check the community-driven
+[examples](https://github.com/front-commerce/examples) repository.
+
+Contribute examples of patterns you proved to be useful in your Front-Commerce
+projects there. The Front-Commerce support team also contributes new examples
+regularly!
+
+# Extensions
+
+These projects extends Front-Commerce core features using Front-Commerce
+extension points.
+
+## Analytics: Facebook conversions API
+
+_Contributor: PH2M_
+
+> Implement the
+> [Facebook conversions API](https://www.facebook.com/business/help/2041148702652965?id=818859032317965)
+> to your Front-Commerce project
+
+[View on Github.](https://github.com/PH2M/front-commerce-facebook-conversions-api)
+
+## External login: Apple
+
+_Contributor: PH2M_
+
+> Implement the external login with Apple your Front-Commerce project using
+> [passport-apple](https://www.npmjs.com/package/passport-apple) and following
+> [Front-Commerce documentation](/docs/2.x/advanced/features/external-logins/)
+
+[View on Github.](https://github.com/PH2M/front-commerce-external-login-apple)
```

Comment:
I wonder if it would not look better in a table format

| Name                                | Contributor | Description                                                                                                                                                                                                                         | Link                                                                               |
|-------------------------------------|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|
| Analytics: Facebook conversions API | PH2M        | Implement the [Facebook conversions API](https://www.facebook.com/business/help/2041148702652965?id=818859032317965) to your Front-Commerce project                                                                                 | [View on Github](https://github.com/PH2M/front-commerce-facebook-conversions-api)  |
| External login: Apple               | PH2M        | Implement the external login with Apple your Front-Commerce project using [passport-apple](https://www.npmjs.com/package/passport-apple) and following [Front-Commerce documentation](/docs/2.x/advanced/features/external-logins/) | [View on Github](https://github.com/PH2M/front-commerce-external-login-apple)      |


```suggestion
| Name                                | Contributor | Description                                                                                                                                                                                                                         | Link                                                                               |
|-------------------------------------|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|
| Analytics: Facebook conversions API | PH2M        | Implement the [Facebook conversions API](https://www.facebook.com/business/help/2041148702652965?id=818859032317965) to your Front-Commerce project                                                                                 | [View on Github](https://github.com/PH2M/front-commerce-facebook-conversions-api)  |
| External login: Apple               | PH2M        | Implement the external login with Apple your Front-Commerce project using [passport-apple](https://www.npmjs.com/package/passport-apple) and following [Front-Commerce documentation](/docs/2.x/advanced/features/external-logins/) | [View on Github](https://github.com/PH2M/front-commerce-external-login-apple)      |
```

Reply:
Maybe even an extra column "Feature" which has `Analytics` and `External login` but that might be too extensive

Reply:
It's great to have a more consistent pattern for each content. But I'm not sure tables are a good move now.

What I love about titles is that we can directly link to an item. In the current state, I tend to favor it more than having a way to have condensed information … because there are not many content (yet!) to display on this page.

Also, table are hard to make responsive and I'm afraid it will lead to ugly designs on mobile.

That's something we could change later however.

Reply:
@paul-vd I reworked the section using a list in c24e37a

It's more compact, responsive but for now anchor links are lost… we could add them back using handcrafted `<a id="test" href="#test">#</a>` later if needed

Reply:
Correct header levels looks much better already, I agree with the table, and I think when we get more we should anyways look at having a "cards list" format, which can also be searchable. But no need to over-engeneer at this stage.